spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -177,7 +177,7 @@ sealed class SpdxExpression {
  *
  * [1]: https://spdx.org/spdx-specification-21-web-version#h.jxpfx0ykyb60
  */
-data class SpdxCompoundExpression(
+class SpdxCompoundExpression(
```

Comment:
This could stay a `data class` and you could still override `equals()` and `hashCode()`. Any reason to *not* keep it as a `data class`?

Reply:
With `equals()`, `hashCode()`, and `toString()` already overridden, the only remaining data class functions are the `componentN` functions and `copy()`. And both do not provide much value, because the SDPX expression classes have only a few properties.